.github/workflows/jira-sync.yml

```
@@ -0,0 +1,76 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+on:
+  issues:
+    types: [labeled, opened, closed, deleted, reopened]
+  issue_comment:
+    types: [created]
+  workflow_dispatch:
+
+name: Jira Issue Sync - test
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    name: Jira Issue sync
+    steps:    
+      - name: Login
+        uses: atlassian/gajira-login@45fd029b9f1d6d8926c6f04175aa80c0e42c9026 # v3.0.1
+        env:
+          JIRA_BASE_URL: ${{ secrets.JIRA_BASE_URL }}
+          JIRA_USER_EMAIL: ${{ secrets.JIRA_USER_EMAIL }}
+          JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
+
+      - name: Set ticket type
+        id: set-ticket-type
+        run: |
+          echo "TYPE=GH Issue" >> $GITHUB_OUTPUT
```

This looks to be hitting a shellcheck lint: https://github.com/hashicorp/nomad/actions/runs/9453086274/job/26037680253?pr=23286 I think we need to quote this?
```suggestion
        run: |
          echo "TYPE=GH Issue" >> "$GITHUB_OUTPUT"
```

---

Thanks!